server/app/app.go

```
@@ -34,7 +38,14 @@ func NewApp(logger *log.Logger, store store.Store, config *config.Config) (*App,
 	if err != nil {
 		return nil, errors.Wrap(err, "can't construct graph from DB")
 	}
-	return &App{logger, store, config, graph}, nil
+
+	environment := make(map[string]string)
+	youtubeAPIKey, ok := os.LookupEnv(YoutubeAPIKey)
+	if ok {
```

Should we return an error when there is no API key?